source/brailleDisplayDrivers/hims.py

```diff
@@ -701,7 +799,7 @@ class KeyInputGesture(braille.BrailleDisplayGesture, brailleInput.BrailleInputGe
 
 	source = BrailleDisplayDriver.name
 
-	def __init__(self, model, keys):
+	def __init__(self, model, keys, isHid=False):

```

Comment:
Please add type hints where possible

```suggestion
	def __init__(self, model, keys, isHid: bool = False):
```